packages/mobile/e2e/src/utils/utils.js

```diff
@@ -58,20 +57,28 @@ export async function skipTo(nextScreen) {
 }
 
 export async function enterPinUi(pin = DEFAULT_PIN) {
-  for (const digit of pin) {
-    try {
-      if (device.getPlatform() === 'ios') {
-        await element(by.id(`digit${digit}`))
-          .atIndex(1)
-          .tap()
-      } else {
-        await element(by.id(`digit${digit}`))
-          .atIndex(0)
-          .tap()
+  try {
+    await device.disableSynchronization()
+    await sleep(250)
```

Comment:
Same comment as before about the `sleep` :(

Reply:
This is just a small sleep between taps of the pin to give the app a second to register the tap. Without this the pin entry on iOS can take upwards of 5 seconds, and this cuts it down to around 1.5 seconds per pin entry. There might be a way to use the waitFor and the dot that appears after a second; however, this would slow down pin entry on iOS and Android.

Reply:
Commented above with attempts to move to `waitFor`s.